editoast/src/views/timetable/stdcm.rs

```diff
@@ -99,10 +99,10 @@ enum StdcmError {
     TrainSimulationFail,
     #[error("Path items are invalid")]
     InvalidPathItems { items: Vec<InvalidPathItem> },
-    #[error("Invalid consist mass: {message}")]
-    InvalidConsistMass { message: String },
-    #[error("Invalid consist length: {message}")]
-    InvalidConsistLength { message: String },
+    #[error("Invalid consist mass: it should be greater than {min}")]
+    InvalidConsistMass { min: f64 },
+    #[error("Invalid consist length: it should be greater than {min}")]
+    InvalidConsistLength { min: f64 },
```

Comment:
```suggestion
    #[error("Invalid consist mass: it should be greater than {min}")]
    InvalidConsistMass { provided_consist_mass: f64, expected_min: f64 },
    #[error("Invalid consist length: it should be greater than {min}")]
    InvalidConsistLength { provided_consist_length: f64, expected_min: f64 },
```

nit: returning the problematic value to the sender may help them to figure out the error faster